docs/development_guide.md

```diff
@@ -167,11 +167,23 @@ TODO:
 
 ## Helm chart
 
-The Open Service Mesh control plane chart is located in the
-[`charts/openservicemesh`](charts/openservicemesh) folder. The [`charts/patch`](charts/patch)
-chart consists of the Open Service Mesh proxy specification, which is used by the proxy
-injector to inject the proxy container. Both charts depend on the partials
-subchart which can be found in the [`charts/partials`](charts/partials) folder.
+The Open Service Mesh control plane charts are located in the
+[`charts/osm`](../charts/osm) folder.
+
+The [`charts/osm/values.yaml`](../charts/osm/values.yaml) file defines the default value for properties
```

Comment:
Would this description about the contents of the chart fit better in the README at the root of the chart?

Reply:
We could do it that way. I was thinking it makes sense to be in the helm section of the development guide. WDYT?

Reply:
Yeah, I think it does make more sense here. If we publish the chart standalone, we wouldn't need those kinds of descriptions.